### DISCUSSION TITLE
Attendance System using Arduino and Laravel

### DISCUSSION THREAD

**Discussion by Migs00** [community/community#139861] [Programming Help]
### Body

So my team is working on a HRIS project and we have RFID as part of the system, problem is, we can't connect the Arduino to Laravel. I have this error on arduino saying error in sending http post request -1, I already checked the api.php and added the post request in there. I also tried it with php only and it works, we just don't know about the Laravel framework. We appreciate the response.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by R-I-T-U** [comment]


**Reply by Migs00** [reply]
Is there a solution for it to work?

**Comment by Migs00** [comment]
I'll give it a try and let you know. Much appreciated!

**Comment by Migs00** [comment]
In app/Http/Middleware/VerifyCsrfToken.php, I don't have this in my project file, Im using laravel 11, probably because I have api.php on my routes.

**Reply by R0b0t-Maker** [reply]
In Laravel, CSRF protection applies to all incoming requests except those that use APIs. Since you're using api.php for routing, the CSRF protection shouldn't apply to those routes by default, which might explain why you don't see VerifyCsrfToken.php.
